test/builder/meta_test.go

```diff
@@ -0,0 +1,48 @@
+package builder_test
+
+import (
+	"testing"
+	"github.com/google/go-cmp/cmp"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	tb "github.com/tektoncd/triggers/test/builder"
+)
+
+func TestObjectMeta(t *testing.T) {
+	// Create struct
+	objectMeta := new(metav1.ObjectMeta)
+	// Modify
+	tb.ObjectMeta(
+		tb.Label("key", "value"),
+		tb.OwnerReference("something", "SomeKind"),
+	)(objectMeta)
+	// Expected stub
```

Comment:
imo this comment and the two above kinda just repeat what the code says so I don't think we need them - I like the idea of only including comments when the code can't be made clear enough on its own to not need them (https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/ seems like a decent representation of this argument)

Reply:
This area was slightly different than the others since I had to create a meta object and then pass it to the object/type meta `modifiers`, which might seem like a builder in comparison to the other files. I thought the first comment _could_ be helpful and then I got carried away. I agree they should be omitted.